TweetMate/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.ybsystem.tweetmate">
+
+    <!-- Permissions -->
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/0a5013e538213bacb2774f24bfa4ed6959fa720c/TweetMate/src/main/AndroidManifest.xml#L8"></span>WRITE_EXTERNAL_STORAGE no longer provides write access when targeting Android 11+, even when using <code>requestLegacyExternalStorage</code></td>
    </tr>
  </tbody>
</table>